stl/inc/xutility

```diff
@@ -5947,10 +5947,13 @@ struct _CXX17_DEPRECATE_ITERATOR_BASE_CLASS iterator { // base type for iterator
 // FUNCTION TEMPLATE _Float_abs_bits
 template <class _Ty, enable_if_t<is_floating_point_v<_Ty>, int> = 0>
 _NODISCARD _CONSTEXPR_BIT_CAST auto _Float_abs_bits(const _Ty& _Xx) {
-    using _Traits    = _Floating_type_traits<_Ty>;
+#if defined(__LDBL_DIG__) && __LDBL_DIG__ == 18
+    using _Traits = _Floating_type_traits<_Ty>;
+#else // ^^^ 80-bit long double (not supported by MSVC in general, see GH-1316) / 64-bit long double vvv

```

Comment:
The convention for specifying comments on the preprocessor conditions would be:
```cpp
// ^^^ defined(__LDBL_DIG__) && __LDBL_DIG__ == 18 ^^^ // vvv defined(__LDBL_DIG__) && __LDBL_DIG__ != 18 vvv
```
But, I think the comment provides helpful information on what the condition is, maybe it needs to be put somewhere else? This probably needs maintainer input.

Reply:
This is actually a fairly common style; we sometimes use a more narrative description of the cases that the conditional is attempting to detect. For example, `^^^ Compiling with clang / some other compiler vvv` conveys more information than `^^^ defined(__clang__) / !defined(__clang__) vvv`.